pkg/action/patch.go

```diff
@@ -18,6 +18,12 @@ func PatchPipelineRun(ctx context.Context, logger *zap.SugaredLogger, whatPatchi
 		return nil, nil
 	}
 	var patchedPR *tektonv1.PipelineRun
+	// double the retry based on recent RHTAP testing
+	doubleRetry := retry.DefaultRetry
+	doubleRetry.Steps *= 2
+	doubleRetry.Duration *= 2
+	doubleRetry.Factor *= 2
+	doubleRetry.Jitter *= 2
```

Comment:
i wonder if we should make this configurable and if there is any downside, do you think you see any?

(nit: let's remove the comment since that would be confusing for the uneducated, or if we really want this we may want to link to the jira url to give more contexts)

Reply:
yeah making it configurable came to mind, I just went for expediency here, as we are blocked in upgrading PaC in RHTAP on this.

if you have a quick/easy pointer for making it configurable, I can try and run with it.

I'll switch the comment to a Jira link in the interim.

Reply:
comment changed

Reply:
it's fine skipping configuration i think , i was just wondering  out loud, introducing a setting come with more cost (doc upstream/downstream, confusion, maint etc...)

let's see what the e2e tests come back with first.>